client/signup/steps/jpo-summary/index.jsx

```diff
@@ -87,30 +68,91 @@ const JPOSummaryStep = React.createClass( {
 
 		const checkMark = <Gridicon
 			className="plan-features__item-checkmark"
-			size={ 18 } icon="checkmark" />;
+			size={ 18 }
+			icon="checkmark"
+			/>;
+
+		const {
+			signupProgress: {
+				0: {
+					jpoSiteTitle
+				},
+				1: {
+					jpoSiteType
+				},
+				2: {
+					jpoHomepage
+				},
+				3: {
+					jpoContactForm
+				}
+			}
+		} = this.props;
+
+		const pathPrefix = '/start/jetpack-onboarding/';
 
 		return (
 			<div className="jpo__summary-wrapper">
 				<table>
 					<tbody>
 						<tr>
 							<td>
-								<div className="jpo__summary-col-header">Steps you've completed:</div>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( "Steps you've completed:" )
+									}
+								</div>
 								<ul className="jpo-summary__completed-onboarding">
-									{ 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle ? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li> : null }
-									{ 'undefined' !== typeof this.props.signupProgress[1].jpoSiteType ? <li>{ checkMark } { translate( 'Type of Site' ) }</li> : null }
-									{ 'undefined' !== typeof this.props.signupProgress[2].jpoHomepage ? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li> : null }
-									{ 'undefined' !== typeof this.props.signupProgress[3].jpoContactForm ? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li> : null }
-									{ connectionToJetpackComplete ? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li> : null }
+									{
+										jpoSiteTitle
+											? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li>
+											: null
+									}
+									{
+										jpoSiteType
+											? <li>{ checkMark } { translate( 'Type of Site' ) }</li>
+											: null
+									}
+									{
+										jpoHomepage
+											? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li>
+											: null
+									}
+									{
+										jpoContactForm
+											? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li>
+											: null
+									}
+									{
+										connectionToJetpackComplete
+											? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li>
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Jetpack Connection' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).